qui/tray/domains.py

```diff
@@ -268,8 +268,8 @@ def update_state(self, state):
             self.name.set_label(self.vm.name)
         self._set_submenu(state)
 
-    def update_stats(self, memory_kb):
-        text = '{0} MB'.format(int(memory_kb)//1024)
+    def update_stats(self, memory_kb, cpu_usage):
+        text = '{0} MB, CPU:{1}%'.format(int(memory_kb)//1024, cpu_usage)
```

Comment:
I think this is a bit hard-ish to read - I'd put it like X MB RAM Y% CPU, so that the order of [number] [unit] [thing] is maintained.
Also, I think the numbers will not necessarily be aligned neatly if CPU usage is above 10% for a VM - maybe some better padding would be a good idea?

Reply:
How about this https://imgur.com/a/NPp47Us

Reply:
I like how it looks, but I'm afraid the smaller letters will be not visible very well on higher DPIs - have you perhaps tested it on higher resolutions?

Reply:
I did test it on 720 & 1080 and it looks good. Another option is to shrink the progress bar height  and increase the font size. Or we can stick with a text representation.

Reply:
I'd keep the bar, if it works fine in 1080p. On bigger resolutions the bar itself will still provide information, so I think this will be good.

Reply:
I will work on that and push the code